_haikus/caterpillar_cloud.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Caterpillar Cloud
+author: Pranav
+---
+
+New Cloud<br>
+With time who will fly soon<br>
+Like a butterfly<br>
+Across the network<br>
```

Comment:
Haikus need to follow a certain poetic format. You can read more about it in this project's `README.md` file. To check that your haiku is meeting the right number of syllables, you can input it into [http://www.haikusyllablecounter.com/](http://www.haikusyllablecounter.com/).

Right now, you have four lines when you should have three. The last two lines are each 5 syllables, so they can be used as either the first line or the last line. Your middle line should have 7 syllables.